src/Raven.Server/Commercial/LicenseManager.cs

```diff
@@ -77,8 +76,6 @@ public class LicenseManager : IDisposable
             FullVersion = ServerVersion.FullVersion
         };
 
-        internal static bool IgnoreProcessorAffinityChanges = false;
```

Comment:
should be static not local. it affects whole process anyway.

Reply:
done